codecov.yml

```diff
@@ -0,0 +1,3 @@
+codecov:
+  notify:
+    after_n_builds: 10
```

Comment:
could you comment on why we might want to wait for 10 builds before reporting results?